src/GlobalErrorHandler.js

```diff
@@ -55,11 +57,15 @@ class GlobalErrorHandler extends React.Component {
 
     return error ? (
       <GlobalErrorScreen>
-        <GenericError
-          detailsTitle={error.message}
-          detailsContent={errorStack}
-          reportCallback={SENTRY_DSN ? this.handleReportClick : null}
-        />
+        {error instanceof DisputeNotFound ? (
```

Comment:
Not sure if we should do this, this is the global error the dispute not found is not a global error is an specific error and should not be handled by the global error which intention is to capture all the errors that were not captured by any specific component error. So here i would suggest to re use the `GlobalErrorScreen` and pass a children with the specific error. But not handle the dispute error inside the global error

Reply:
Not sure i follow. I think that this component is not meant to only capture unhandled generic errors but also any error that we would like to handle (i take that by `Global` doesn't reference a generic error but the error hub of the app). Doesn't make sense to me to pass another component as children that will use the `GlobalErrorScreen` when we are already using it here.

Reply:
you don't need the globalErrorHandler at all this is a dispute detail error the same than any of the dispute detail error, think about if not we are going to have some dispute errors here another dispute details error inside the dispute detail component, what you need is the `<GlobalErrorScreen/>` not the handler, also i think that you created the `disputeNotFound` inside a generic error folder instead inside the `Dispute` Folder as the other dispute errors

Reply:
take a look [here](https://reactjs.org/docs/error-boundaries.html) if you want to what is the idea for this component

Reply:
I think we do need the global error handler. We need to treat this error different because we are going to display it in a [different way](https://github.com/aragon/court-dashboard/pull/198#discussion_r385010775) than the other types of dispute errors that are local to the dispute detail. Not sure if you are suggesting to use  `<GlobalErrorScreen />`  inside the dispute detail, but that would not work because it would render as a child of `<MainView />`.

Reply:
ohh is a full screen error this one! so yes you are right, so let's change the dispute error inside the dispute component folder maybe?

Reply:
I think that since it's a global error and it's not a component that will be a rendered as a child of the main `Disputes` component it may make more sense to leave it in the generic `errors` folder what do you think ?

Reply:
mmm but is not a global error for the whole app, is an error only for disputes so i think that should be in the ddispute folder

Reply:
Yeah okay that's also true.